BusBoard.ConsoleApp/Program.cs

```diff
@@ -21,8 +19,8 @@ static void Main(string[] args)
                 Console.WriteLine("Thank you for using BusBoard");
                 break;
             }
-            var buses = Helpers.GetBuses(input, client);
-            var topFive = buses.Take(5).Select(o => o.GetBus());
+            var buses = tflApiHelper.GetBusData(input);
+            var topFive = buses.Take(5).Select(bus => bus.GetBus());
```

Comment:
Should be `GetBusData`? Remember to check that your program works before pushing new changes 🙂

Reply:
GetBusData is the tflApiHelper method and GetBus is the BusEntry method. This code works but I realise the naming should be the other way round i.e. GetBuses is the tflApiHelper method (as we're getting a list of buses) and GetBusData  is the BusEntry method. Will fix this :)